xlators/cluster/dht/src/dht-common.h

```diff
@@ -64,6 +64,19 @@ typedef int (*dht_refresh_layout_unlock)(call_frame_t *frame, xlator_t *this,
 
 typedef int (*dht_refresh_layout_done_handle)(call_frame_t *frame);
 
+struct dht_layout_entry {
+    int err; /* 0 = normal
+                -1 = dir exists and no xattr
+                >0 = dir lookup failed with errno
+             */
+    uint32_t start;
+    uint32_t stop;
+    uint32_t commit_hash;
+    xlator_t *xlator;
```

Comment:
nit: can you please ensure correct padding, as much as possible, so 8 bytes vars appear before 4 byte, etc.?

Reply:
Hmm... what's the point on doing `8 bytes vars appear before 4 bytes`? This structure should be packed/aligned/padded/whatever  well enough on both 32- and 64-bit ABIs.

Reply:
Ah, did not see the int err in the beginning - sorry, it looks good and aligned (we don't do much packing in Gluster, only in some structs, not sure why).